press_kit.md

```diff
@@ -0,0 +1,174 @@
+﻿
+# **PRESS KIT**
+
+![](https://raw.githubusercontent.com/Barbanio/Press-Kit/main/picture_press_kit.webp)
+
+## **INDEX:**
+
+**[1. AsyncAPI](#_heading=h.40s9q7rdmzv1)**
+
+[1.1. Open governance model](#_heading=h.m14n0k7zs2qm) 
+
+[1.2. AsyncAPI, in numbers](#_heading=h.f8xlcwy3vi6) 
+
+**[2. History of the project](#_heading=h.5yg403h4t8gi)**
+
+**[3. Values of AsyncAPI](#_heading=h.4me7screlap)**
+
+**[4. Brands and companies using AsyncAPI](#_heading=h.xxo7nwvlczto)**
+
+**[5. Funding](#_heading=h.bx8fkbp32vqc)**
+
+**[6. Contact details](#_heading=h.7oacawrz9cjj)**
+
+**[7. Frequently asked questions](#_heading=h.9m5tif8t2hnq)**

```

Comment:
These links are not working. Also, I'd remove the bold format from them.

I think the best thing you can do here is to remove the numbers and use an ordered list:

```md
1. [AsyncAPI](#asyncapi)
    1. [Open governance model](#open-governance-model)
    1. [AsyncAPI, in numbers](#asyncapi-in-numbers)
1. [History of the project](#history-of-the-project)
etc.
```